backend/qemu.pm

```diff
@@ -876,7 +876,7 @@ sub start_qemu ($self) {
             }
         }
 
-        sp('enable-kvm') unless $vars->{QEMU_NO_KVM};
+        sp('enable-kvm') if -r '/dev/kvm' && !$vars->{QEMU_NO_KVM};
```

Comment:
There was https://github.com/os-autoinst/os-autoinst/pull/1890#discussion_r773273819 "So what happens if there's no /dev/kvm device? It gets switched off silently?"

To give the answer here: "Yes" :)

I am wondering, maybe we should keep it more explicit

Reply:
My issue is its being coupled with `QEMU_NO_KVM`. I would not be opposed to automatically enabling or disabling kvm (and making sure it's clear from the logs).

Reply:
In other words, we have a very explicit variable and now you'e adding heuristics to it. This feels like we're asking for confusion.

Reply:
It would be a lot easier if we had something like `QEMU_KVM=auto`

Reply:
yes. I guess you are right

Reply:
but wait, I am only changing the default case when the variable `QEMU_NO_KVM` isn't set.